{"ID": "0c02a522-e71e-4bf4-b500-25fdbb0aa663", "effective_cve": "CVE-2017-17522", "image": {"exact": "docker.io/cloudbees/cloudbees-core-oc@sha256:9cd85ee84e401dc27e3a8268aae67b594a651b2f4c7fc056ca14c7b0a0a6b82d"}, "label": "??", "package": {"name": "python2.7", "version": "2.7.16-2+deb10u1"}, "timestamp": "2023-01-26T16:26:19+00:00", "tool": "grype@v0.55.0-24-gc01ee9b", "user": "westonsteimel", "vulnerability_id": "CVE-2017-17522"}